grunt_tasks/fontello.js

```diff
@@ -45,13 +46,14 @@ module.exports = function (grunt) {
     } else {
         // Download font archive from data.kitware.com
         grunt.config.merge({
-            shell: {
+            curl: {
                 fontello: {
-                    command: 'curl "' + srcUrl + '" -o "' + archivePath + '"'
```

Comment:
We actually used `grunt-curl` before, but we changed it due to flaky downloads.  See the commit here: https://github.com/girder/girder/commit/051cf2b881f9f628546f53a93983841ee9d6f534.